apps/components-e2e/src/components/drawer/drawer.e2e.ts

```diff
@@ -39,88 +44,100 @@ const scrollUp = ClientFunction(distance => {
   window.scrollBy(0, -distance);
 });
 
-fixture('Drawer').page('http://localhost:4200/drawer');
+fixture('Drawer')
+  .only.page('http://localhost:4200/drawer')
```

Comment:
@ffriedl89 you should not have to merge it I forgot an .only inside. Please fix it.